src/utils/fs.ts

```diff
@@ -86,6 +88,16 @@ const resolveFunctionsDirectories = (input: string | string[]) => {
   return absoluteDirectories
 }
 
+const mkdirAndWriteFile: typeof pWriteFile = async (path, ...params) => {
+  if (typeof path === 'string') {
+    const directory = dirname(path)
+
+    await makeDir(directory)
```

Comment:
why can we not use the native fs `mkdirSync(directory, { recursive: true })` with the recursive option as this was introduced in Node.js 10 and should be available no need for async and an own library like `make-dir`

Reply:
+1 to this :arrow_up: